Fixes #3964 - Gloucestershire Disused Taxiway SMR Change

SwietyMik opened the pull request:
Fixes #3964 

# Summary of changes

Disused Taxiway F, separated and changed SMR appearance instead of removing it completely
Updated taxiway markings due taxiway removal - including centerline and holding points

# Screenshots (if necessary)

![image](https://user-images.githubusercontent.com/69194446/148649306-6bdb0b14-a883-445d-b80f-01c2c326fff0.png)

SwietyMik commented:
couldnt find a better suitable colour, didn't want to use CCs disused as thats something that would strain ur eyes, whoever made it that bright smh

hsugden reviewed (commented):
smrGrey is what we use at, for example, Doncaster. How does it look with that?

SwietyMik commented:
> smrGrey is what we use at, for example, Doncaster. How does it look with that?

Much better @hsugden Cheers
![image](https://user-images.githubusercontent.com/69194446/150642803-645593fb-0827-4ef5-bc69-9626e7001517.png)

GeekPro101 reviewed (changes requested):
Being reallyyyy pernickety, it looks like the closure goes slightly further along:
![image](https://user-images.githubusercontent.com/14115426/150654674-76665016-d69f-4276-8af2-dba2837c4e06.png)

SwietyMik commented:
😶

SwietyMik commented:
![image](https://user-images.githubusercontent.com/69194446/150658476-1470222a-3e4c-4007-98ca-8856f19dcddb.png) @GeekPro101 
Updated the adjacent polygons,
- Disused F
- Taxiway E
- Runway 22/04
Updated taxi-lines,
- RWY22 Leadup
- Taxiway F Centerline (curve)
- Taxiway E

GeekPro101 reviewed (commented):
This might be something better left for a general cleanup, but I don't see the point in having several KMZs at one point - there's the Gloucestershire one, the 2106 one, and now a 2201 one
